runtime/kusama/src/lib.rs

```diff
@@ -1979,6 +2025,7 @@ sp_api::impl_runtime_apis! {
 			list_benchmark!(list, extra, pallet_treasury, Treasury);
 			list_benchmark!(list, extra, pallet_utility, Utility);
 			list_benchmark!(list, extra, pallet_vesting, Vesting);
+			list_benchmark!(list, extra, pallet_bags_list, BagsList);
```

Comment:
would prefer this is in alphabetical order

Reply:
4482a6d7fcffc30b241b5e4d520ad224bbd7742e